lib/ui/myplot/new_my_plot_child_item.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:farmsmart_flutter/data/model/crop_entity.dart';
+import 'package:farmsmart_flutter/utils/colors.dart';
+import 'package:farmsmart_flutter/utils/dimens.dart';
+import 'package:farmsmart_flutter/utils/strings.dart';
+import 'package:flutter/material.dart';
+import 'package:farmsmart_flutter/ui/common/network_image_from_future.dart';
+import 'my_plot_page.dart';
+
+class MyNewPlotListItem {
+  Widget buildListItem(CropEntity cropsData, HomeMyPlotPageStyle myPlotStyle) {
+
+    //FIXME: This variables should be with the cropData, but for the moment are hardcored coz of the CMS Data.
+    const DayText = "Day 6";
+    const currentStage = "Planting";
+
+    return Column(
+      children: <Widget>[
+        Container(
+          padding: myPlotStyle.listPadding,
+          child: Row(
+            mainAxisAlignment: myPlotStyle.mainAxisAlignmentSpaceBeetwen,
+            children: <Widget>[
+              Column(
+                crossAxisAlignment: myPlotStyle.crossAxisAlignmentStart,
+                children: <Widget>[
+                  Row(
+                    children: <Widget>[
+                      Text(
+                          cropsData.name ?? Strings.defaultCropNameText,
+                          style: myPlotStyle.cropNameTextStyle),
+                      Padding(
+                        padding: myPlotStyle.paddingBetweenElements,
+                      )
+                    ],
+                  ),
+                  Row(
+                    children: <Widget>[
+                    Text(currentStage, style: myPlotStyle.cropStatusTextStyle),
+                    Padding(
+                      padding: myPlotStyle.paddingBeforeCropDayCount,
+                    )
+                  ],
+                  ),
+                  Row(
+                    children: <Widget>[
+                    Container(
+                      padding: myPlotStyle.paddingForDayCount,
+                      decoration: BoxDecoration(
+                        color: myPlotStyle.primaryGreen.withOpacity(myPlotStyle.oppacityForDayCount),
```

Comment:
can we just have a colour for that feature (that includes the opacity?) also name it in line with the generic view model so something like "detailTextBackgroundColor", since in the new view model this field is named "detailText"

Reply:
For the moment i dont found how to define the Color + oppacity correctly in Style class.